frontend-student/src/components/about/About.jsx

```
@@ -736,18 +736,11 @@ const AboutUs = () => {
-              >
-                Start Your Free Trial
-              </a>
-              <a
-                href='#'
-                className='px-8 py-4 rounded-full bg-transparent text-white font-medium border border-white hover:bg-blue-700 transition duration-300 text-lg'
-              >
-                Schedule a Demo
-              </a>
+              <Link to='/signup'>
+                <button className='px-8 py-3 bg-white text-blue-600 font-bold rounded-lg hover:bg-gray-100 transition'>
+                  Sign Up Now
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)